css/index.css-copy.css

```diff
@@ -0,0 +1,433 @@
+ margin: 0;
+ padding: 0;
+ border: 0;
+ font-size: 100%;
+ font: inherit;
+ vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+ display: block;
+}
+body {
+ line-height: 1;
+}
+ol, ul {
+ list-style: none;
+}
+blockquote, q {
+ quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+ content: '';
+ content: none;
+}
+table {
+ border-collapse: collapse;
+ border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+ box-sizing: border-box;
+}
+
+html, body {
+ height: 100%;
+ font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+ font-family: 'Bangers', cursive;
+ letter-spacing: 1px;
+ margin-bottom: 15px;
+}
+
+/*----------------------------------------------------------------------------*/
+
+.container {
+    margin: auto;
+    width: 60%;
+    font-family: 'Exo 2', sans-serif;
+}
+
+/* At 800px And below do the following: Tablet */
+@media (max-width: 800px) {
+    .container { 
+        width: 800px;
+    }
+}
+
+/* At 500px and below do the following: Phone  */
+@media (max-width: 500px) {
+    .container {
+        width: 500px
+    }
+}
+
+header {
+    box-sizing: border-box;
+    border-bottom: 2px solid black;
+    padding-bottom: 70px;
+    text-align: center;
+
+}
+
+/* At 800px and below do the following: Tablet */
+@media (max-width: 800px) {
+    header {
+        width: 700px;
+        margin: auto;
+}
+}
+
+.logo {
+    background-color: white;
+
+}
+/* At 800px and below do the following: Tablet */
+@media (max-width: 800px) {
+    .logo {
+        display: flex;
+        flex-direction: reverse; 
+        margin-top: -90px;
+        margin-left: 259px;
+
+    }
+}
+
+@media (max-width: 500px) {
+    .logo {
+       flex-direction: reverse; 
+    }
+}
+
+nav {
+    word-spacing: 40px; 
+    flex-direction: row;
+    padding: 10px;
+
+}
+
+@media (max-width: 800px) {
+nav {
+    word-spacing: 40px;
+    margin-top: 90px;
+}
+}
+
+/* At 500px And below do the following: */
+@media (max-width: 500px) {
+    nav {
+   
+        margin-top: 100px;
+   
+   
+    }
+   }
+   
+   
+   
+   nav a {
+    text-decoration: none;
+    color: #b3b3b3; 
+   
+   }
+   /* At 500px And below do the following: */
+   @media (max-width: 500px) {
+    nav a {
+        display: block ;
+        padding: 20px; 
+        background: white;
+        border-bottom: 1px solid black;
+       
+    }
+   }
+   
+   
+   
+   .top-section {
+   padding-bottom: 10px; 
+   box-sizing: border-box;
+   }
+   
+   
+   
+   .main {
+    box-sizing: border-box;
+    display: inline-block;
+    width: 270px;
+    height: 300px; 
+    text-align: center;
+    font-size: 80px;
+     
+   
+   }
+   
+   /* At 500px And below do the following: */
+   @media (max-width: 500px) {
+    .title {
+        margin-top: -110px; 
+        margin-left: 0px; 
+        width: 230px; 
```

Comment:
I wouldn't hard code width like this, either i'd use `max-width` or use `%` instead of `px`